src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphEntityDAOSpec.scala

```diff
@@ -158,34 +161,50 @@ class GraphEntityDAOSpec extends FlatSpec with Matchers with OrientDbTestFixture
     }
   }
 
-  it should "save updates to an existing entity" in withDefaultTestDatabase { dataSource =>
+  it should "save updates to an existing entity" in withCustomTestDatabase(testData, lwEdges=false) { dataSource =>
```

Comment:
`graph.getEdges` returns an empty list when graphs have lightweight edges, because edge data is stored on the vertices instead. In order for this test to check that saving objects doesn't obliterate edges, the database needs to be stood up with heavyweight edges.